doc-examples/jdbc-example-java/src/main/java/example/CustomEntityRepository.java

```diff
@@ -1,9 +1,17 @@
 package example;
 
 import io.micronaut.data.jdbc.annotation.JdbcRepository;
+import io.micronaut.data.model.Page;
+import io.micronaut.data.model.Pageable;
+import io.micronaut.data.model.Slice;
 import io.micronaut.data.model.query.builder.sql.Dialect;
 import io.micronaut.data.repository.CrudRepository;
 
+import java.util.List;
+
 @JdbcRepository(dialect = Dialect.H2)
 public interface CustomEntityRepository extends CrudRepository<CustomEntity, Long> {
+    Slice<CustomEntity> findAll(Pageable pageable);
+
+    Page<CustomEntity> findByNameIn(List<String> names, Pageable pageable);
```

Comment:
Can you please add an example of a custom query with IN that does have an expression

Reply:
Added https://github.com/micronaut-projects/micronaut-data/pull/3245/commits/03324f5cd93aa14de47c310ecc4910912a46e558

Reply:
It might fix this issue as well https://github.com/micronaut-projects/micronaut-data/issues/2959